### PR TITLE
Use correct channel information

### LINKED ISSUE #2385: After installation no default channel is used
@MorrisJobke I suppose we should default to `stable` then? 

screen shot 2016-11-28 at 18 20 38

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @rullzer @LukasReschke 

fixes #2385

### CODE REVIEW

**Comment by nickvergessen** [comment]
👍

File: version.php [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
 $OC_VersionCanBeUpgradedFrom = array(9, 1);
 
-// The ownCloud channel
+// default Nextcloud channel
```
Comment (nickvergessen) [inline version.php]: I hope this doesnt break packaging? 🙊

Comment (LukasReschke) [inline version.php]: Should be fine. Just don't change the line below ever ;)

**Comment by rullzer** [comment]
LGTM

**Pull request merged**